open-liberty/content.md

```diff
@@ -49,17 +49,20 @@ Please note that this pattern will duplicate the docker layers for those artifac
 
 There are multiple tags available in this repository.
 
-The `kernel` image contains just the Liberty kernel and no additional runtime features. This image is the recommended basis for custom built images, so that they can contain only the features required for a specific application. For example, the following Dockerfile starts with this image, copies in the `server.xml` that lists the features required by the application, and then uses the `configure.sh` script to download those features from the online repository.
+The `kernel-slim` image contains just the Liberty kernel and no additional runtime features. This image is the recommended basis for custom built images, so that they can contain only the features required for a specific application. For example, the following Dockerfile starts with this image, copies in the `server.xml` that lists the features required by the application, and then uses the `configure.sh` script to download those features from the online repository.
 
 ```dockerfile
-FROM %%IMAGE%%:kernel
+FROM %%IMAGE%%:kernel-slim
 COPY --chown=1001:0  Sample1.war /config/dropins/
 COPY --chown=1001:0  server.xml /config/
 RUN configure.sh
 ```
 
 The full list of images are found in the `Supported tags and respective Dockerfile links` section above.
 
+Note: As of the 20.0.0.11 release the kernel images are deprecated and will not the updated. The kernel-slim images will replace them in this and future releases.
+Please visit https://github.com/OpenLiberty/ci.docker#building-an-application-image for more information on how to expand the kernel-slim image to contain the features needed.
```

Comment:
As seen in https://github.com/docker-library/docs/pull/1838/checks?check_run_id=1426750446#step:4:17, `markdownfmt` expects this to be one line (or to have more whitespace to create either a line break or a paragraph break, depending on what you intended):

```suggestion
Note: As of the 20.0.0.11 release the kernel images are deprecated and will not the updated. The kernel-slim images will replace them in this and future releases. Please visit https://github.com/OpenLiberty/ci.docker#building-an-application-image for more information on how to expand the kernel-slim image to contain the features needed.
```